src/test/java/domain/game/JanggiGameTest.java

```diff
@@ -0,0 +1,33 @@
+package domain.game;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import domain.board.Board;
+import domain.board.BoardLocation;
+import domain.piece.Piece;
+import domain.piece.Team;
+import fixture.BoardFixture;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class JanggiGameTest {
+
+    @Test
+    @DisplayName("이동가능한 좌표로 이동한다면 위치를 변경한다")
+    void process() {
+        //given
+        Map<BoardLocation, Piece> pieces = BoardFixture.createHanTeamBoard();
+        Board board = new Board(pieces);
```

Comment:
어짜피 만들고 싶었던 Fixture는 Board였던 것 같은데, BoardFixture를 통해서 Board 객체를 생성하면 되지 않을까요? 🤔 

```suggestion
        Board board = BoardFixture.create();
        
```

Reply:
맞습니다! 어차피 결국 모두 Board를 만들기 위해 사용되어서 Board를 생성하는 것으로 수정하였습니다!

a0084902b38fb8d25c612f9187dc5997ce0bf4d2